docs/pages/getting-started.html

```diff
@@ -9,7 +9,7 @@
 
 <h3 class="usa-heading">For developers</h3>
 
-<p>The UI components are built on a solid <strong>HTML</strong> foundation, progressively enhanced to provide core experiences across browsers. All users get critical information and experiences. New browsers get the prettiest experiences, while older browsers get less pretty, but usable ones. If JavaScript fails, users will still get a robust HTML foundation.</p>
+<p>The UI Components are built on a solid <strong>HTML</strong> foundation, progressively enhanced to provide core experiences across browsers. All users get critical information and experiences. New browsers get the prettiest experiences, while older browsers get less pretty, but usable ones. If JavaScript fails, users will still get a robust HTML foundation.</p>
```

Comment:
I really don't think we should ever use the word 'pretty' in reference to design, let alone say something we're offering is 'less pretty.' Can we recast this sentence to put things in a more positive light?